src/com/wakatime/intellij/plugin/Dependencies.java

```diff
@@ -47,6 +47,13 @@ public static boolean isPythonInstalled() {
 
     public static String getResourcesLocation() {
         if (Dependencies.resourcesLocation == null) {
+            if (System.getenv("WAKATIME_HOME")) {
+                File appDataFolder = new File(System.getenv("WAKATIME_HOME"));
```

Comment:
We don't need to define `appDataFolder` here, it's only used to find the window's data folder when `WAKATIME_HOME` isn't defined.

Reply:
I've removed it in a separate commit, and will release a new version soon (probably within a few days). Thanks!

Reply:
Wow, that was quick! Thanks again for the awesome project.

Reply:
It's already been merged.

Reply:
But that merge hasn't been put into a new version right?
![image](https://user-images.githubusercontent.com/5614134/36803221-cd901d7c-1c6b-11e8-8c68-8ebcea6807e7.png)

Reply:
Oh, right! Just released a new version 9.0.2 with your change.